crates/viewer/re_data_ui/src/image.rs

```diff
@@ -393,19 +377,13 @@ pub fn tensor_summary_ui_grid_contents(
         | TensorBuffer::F16(_)
         | TensorBuffer::F32(_)
         | TensorBuffer::F64(_) => {}
-        TensorBuffer::Jpeg(jpeg_bytes) => {
```

Comment:
this makes me realize that there's no custom data ui for `EncodedImage` yet unless I missed a blob of code along the way.
Needs to be put on the todo list! 

Edit: Ah this is a much deeper problem obviously. There's no `EncodedImage` component, so this whole line of work makes us loose the ability to see images in the property view. We may want to come up with something new here short or long term. Worth discussing today or later this week :)
![image](https://github.com/user-attachments/assets/99e5676a-245a-4ba4-99b6-5db46c5443a8)